cape/cape_result.py

```diff
@@ -2215,8 +2264,9 @@ def process_buffers(
             if table_row not in buffer_body and count_per_source_per_process < BUFFER_ROW_LIMIT_PER_SOURCE_PER_PROCESS:
                 buffer_body.append(table_row)
                 count_per_source_per_process += 1
-
+            buffers.append(table_row)
```

Comment:
You shouldn't be adding `table_row` here because that is a dictionary. We just want the `buffer` value in `buffers`/`network_buffers`.